app/src/pages/inside/common/launchSuiteGrid/launchSuiteGrid.jsx

```diff
@@ -215,308 +219,264 @@ TiColumn.propTypes = {
-      withHamburger,
-      onEditItem,
-      onDeleteItem,
-      onMove,
-      onForceFinish,
-      onAnalysis,
-      onPatternAnalysis,
-    } = this.props;
-    const hamburgerColumn = {
-      component: HamburgerColumn,
-      customProps: {
-        onDeleteItem,
-        onMove,
-        onForceFinish,
-        onAnalysis,
-        onPatternAnalysis,
-      },
-    };
-    const columns = [
-      {
-        id: ENTITY_NAME,
-        title: {
-          full: 'name',
-          short: 'name',
-        },
-        maxHeight: 170,
-        component: NameColumn,
-        sortable: true,
-        withFilter: true,
-        filterEventInfo: events.NAME_FILTER,
-        customProps: {
-          onEditItem,
-          onClickAttribute: this.handleAttributeFilterClick,
-          onOwnerClick: this.handleOwnerFilterClick,
-          events,
-          withExtensions: withHamburger, // Use extensions for launch level only
-        },
-        sortingEventInfo: events.NAME_SORTING,
-      },
-      {
-        id: ENTITY_START_TIME,
-        title: {
-          full: 'start time',
-          short: 'start',
-        },
-        component: StartTimeColumn,
-        sortable: true,
-        withFilter: true,
-        filterEventInfo: events.START_TIME_FILTER,
-        sortingEventInfo: events.START_TIME_SORTING,
-      },
-      {
-        id: STATS_TOTAL,
-        title: {
-          full: 'total',
-          short: 'ttl',
-        },
-        component: TotalColumn,
-        sortable: true,
-        withFilter: true,
-        filterEventInfo: events.TOTAL_FILTER,
-        sortingEventInfo: events.TOTAL_SORTING,
+    const handleAttributeFilterClick = useCallback(
+      (attribute) => {
+        onFilterClick(
+          [
+            {
+              id: ENTITY_ATTRIBUTE,
+              value: {
+                filteringField: ENTITY_ATTRIBUTE,
+                condition: CONDITION_HAS,
+                value: formatAttribute(attribute),
+              },
+            },
+          ],
+          true,
+        );
+
+        if (events.CLICK_ATTRIBUTES) {
+          trackEvent(events.CLICK_ATTRIBUTES);
+        }
       },
-      {
-        id: STATS_PASSED,
-        title: {
-          full: 'passed',
-          short: 'ps',
-        },
-        component: PassedColumn,
-        sortable: true,
-        withFilter: true,
-        filterEventInfo: events.PASSED_FILTER,
-        sortingEventInfo: events.PASSED_SORTING,
+      [onFilterClick, events, trackEvent],
+    );
+
+    const handleOwnerFilterClick = useCallback(
+      (owner) => {
+        onFilterClick({
+          id: ENTITY_USER,
+          value: {
+            filteringField: ENTITY_NAME,
+            condition: CONDITION_IN,
+            value: owner || '',
+          },
+        });
       },
-      {
-        id: STATS_FAILED,
-        title: {
-          full: 'failed',
-          short: 'fl',
+      [onFilterClick],
+    );
+
+    const getColumns = useMemo(() => {
```

Comment:
It is not a function anymore, so let's update the name:
```suggestion
    const columns= useMemo(() => {
```